src/jit/codegenxarch.cpp

```diff
@@ -2645,8 +2576,6 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
 bool
 CodeGen::genStoreRegisterReturnInLclVar(GenTreePtr treeNode)
 {
-
-
     if (varTypeIsStruct(treeNode))
```

Comment:
If i understand correctly, this case will occur when the return value of a call is a struct returned in two registers.  From what i see in lowerxarch.cpp of GT_CALL, i don't see we are modeling struct return in two registers or a ret-buf return in RAX.  Help me understand how are we modelling the register requirement that the call would be generating a return value in two registers or etbuf in rax?